library/src/androidTest/java/org/xmtp/android/library/HistorySyncTest.kt

```diff
@@ -39,113 +37,108 @@ class HistorySyncTest {
         caroWallet = fixtures.caroAccount
         caro = fixtures.caro
 
+        alixClient = fixtures.alixClient
         boClient = fixtures.boClient
         caroClient = fixtures.caroClient
-        alixClient = fixtures.alixClient
-
-        alixGroup = runBlocking { alixClient.conversations.newGroup(listOf(boClient.inboxId)) }
+    }
 
-        alixClient2 = runBlocking {
-            Client.create(
-                account = alixWallet,
-                options = ClientOptions(
-                    ClientOptions.Api(XMTPEnvironment.LOCAL, false),
-                    appContext = fixtures.context,
-                    dbEncryptionKey = fixtures.key,
-                    dbDirectory = fixtures.context.filesDir.absolutePath.toString()
-                )
+    @Test
+    fun testSyncConsent() = runBlocking {
```

Comment:
Not blocking the release but I cannot get this test passing no matter what I try.